core/src/main/scala/scalaz/EitherT.scala

```diff
@@ -287,6 +287,39 @@ object EitherT extends EitherTInstances {
 
 }
 
+object EitherTNondeterminism {
+  implicit def eitherTNondeterminism[F[_], L](implicit F0: Nondeterminism[F]): Nondeterminism[EitherT[F, L, ?]] =
+    new Nondeterminism[EitherT[F, L, ?]] {
+      val F = Nondeterminism[F]
+
+      override def chooseAny[A](
+                                 head: DisjunctionT[F, L, A],
```

Comment:
The name `EitherT` is used throughout the file; please stick with the standard name.